exercises/practice/reverse-string/.approaches/introduction.md

```diff
@@ -0,0 +1,82 @@
+# Introduction
+
+Reversing a string is quite a common problem in programming, there are many ways to achieve it in PowerShell.
+We can use a built-in `reverse` method for array.
+We can use `StringBuilder` object to create a new string.
+Or we can manipulate the char array before returning it into the reversed string.
+
+
+## General guidance
+
+The goal of this exercise is to reverse a string.
```

Comment:
Not sure this offers a lot of value. The idea of the general guidance part is to list things that apply to _all_ approaches. E.g. the C# approach introduction for `reverse-string` has general guidance on how strings are immutable.